client/src/main/java/com/kritsit/casetracker/client/domain/ui/controller/EditorController.java

```diff
@@ -199,12 +216,7 @@ private void setCalendarColumnWidth(TableColumn column) {
     }
 
     private void setCellValueFactory(TableColumn<List<Day>, String> column, final int dayIndex) {
-        column.setCellValueFactory(new Callback<CellDataFeatures<List<Day>, String>, ObservableValue<String>>() {
-            public ObservableValue<String> call(CellDataFeatures<List<Day>, String> week) {
-                Day day = week.getValue().get(dayIndex);
-                return new SimpleStringProperty(day.toString());
-            }
-        });
+        column.setCellValueFactory(new CellValueFactoryDayCallback(dayIndex));
```

Comment:
This possibly may be avoided with lambdas, as this was the sort of situation that they were designed for. If not then I agree with simply suppressing these warnings